<launch>
    <include file="pid_controls.launch" />
    <node name="maintain_depth" pkg="controls" type="maintain_depth.py" respawn="false" output="screen">
        <param name="desired_depth" type="double" value="5" />
    </node>
</launch>
